test/unit/Fixture/TraitFixture.php

```diff
@@ -135,3 +135,20 @@ class TraitFixtureF
     }
 
 }
+
+trait Trait1FixtureG {
```

Comment:
IMO worth moving to a string source locator within the test: it's only used within that one test, and shouldn't be mixed with the rest

Reply:
Done.